### DISCUSSION TITLE
Welcome to whosonfirst-data Discussions!

### DISCUSSION THREAD

**Discussion by thisisaaronland** [whosonfirst-data/whosonfirst-data#1957] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 622333371** [comment]
Bonjour je suis nouveau dans cette communauté et je souhaite approfondir ma connaissance auprès des vous et apprendre à contribuer merci cordialement

**Reply by nvkelso** [reply]
> Hello I am new in this community and I wish to deepen my knowledge with you and learn to contribute thank you cordially

`Heureux de votre participation :)` -- Happy to for your to join :)

**Comment by Oleg85gg** [comment]
, 👋
